src/OeUI/OemrUI.php

```diff
@@ -499,6 +499,8 @@ private function headerActionJs($arrAction = array())
         $action_top_js = <<<SHWTOP
         <script>
         $(function () {
+            let element = document.querySelector('a#show_hide i');
+            let elementIcon = document.querySelector('a#show_hide i');
```

Comment:
@robertdown 
Depending on the initial environment state will determine if this is needed or not.
I think once local storage is set but I stopped trying to follow the whole OemrUI logic.
Declaring the var to be more global for where it is needed is the fix.

Reply:
![CzLY2Xbq](https://user-images.githubusercontent.com/5618372/212327963-4e6d0909-be3e-426f-b0be-45496436801d.gif)